automation/taskcluster/decision_task.py

```diff
@@ -168,13 +205,18 @@ def release(channel, is_staging, version_name):
         ordered_groups_of_tasks = raptor(result.staging)
     elif command == 'nightly':
         formatted_date = datetime.datetime.now().strftime('%y%V')
-        ordered_groups_of_tasks = release('nightly', result.staging, '1.0.{}'.format(formatted_date))
-    elif command == 'beta':
-        semver = re.compile(r'^v\d+\.\d+\.\d+-beta\.\d+$')
-        if not semver.match(result.tag):
-            raise ValueError('Github tag must be in beta semver format and prefixed with a "v", e.g.: "v1.0.0-beta.0"')
+        ordered_groups_of_tasks = nightly_to_production_app(result.staging, '1.0.{}'.format(formatted_date))
+    elif command == 'github-release':
         version = result.tag[1:]  # remove prefixed "v"
-        ordered_groups_of_tasks = release('beta', False, version)
+        beta_semver = re.compile(r'^v\d+\.\d+\.\d+-beta\.\d+$')
```

Comment:
Follow up: Move these regexes to https://github.com/mozilla-releng/mozilla-version/ and use this library.

Reply:
Oh, I haven't seen this library before - what does it help with? When is it used?
Since `fenix` is deployed to users, and isn't a library consumed by other applications, what use cases would there be to have the version information shared?

Reply:
This lib is meant to avoid duplication of regexes when it comes to version numbers. Over the years, there are so many different places where we try to guess if a Firefox build is a nightly/beta/release based on its version number. For instance: in buildhub, in soccorro, in mozapkpublisher, etc. Therefore, even though fenix is an end-user project, it may be monitored by other projects which would benefit from these regexes.

In some of these projects, we've seen regexes that were too simple and didn't scale well. mozilla-version sets them (hopefully once and for all) so satellite projects don't have to rewrite them.

Reply:
I've created #3312